apis/config/v1/types.go

```diff
@@ -277,3 +277,22 @@ type SySchedArgs struct {
 	// CR name of the default profile for all system calls
 	DefaultProfileName *string `json:"defaultProfileName,omitempty"`
 }
+
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// PeaksArgs holds arguments used to configure the Peaks plugin
+type PeaksArgs struct {
+	metav1.TypeMeta `json:",inline"`
+
+	// Address of load watcher service
+	WatcherAddress string                `json:watcherAddress",inline"`
+	NodePowerModel map[string]PowerModel `json:nodePowerModel",inline"`
```

Comment:
can you add a comment mentioning that you will be refactoring this in a follow-up PR? https://github.com/kubernetes-sigs/scheduler-plugins/pull/829#discussion_r1835548516

Reply:
Added comment on the followup PR feature